### PR TITLE
final project

### PR DESCRIPTION
**Pull request by nancy43**

### CODE REVIEW

**Comment by caiodias** [comment]
Please organize your project files under a new folder with your github user name. Like **student**, student/css, student/js

**Comment by nancy43** [comment]
ok

**Comment by caiodias** [comment]
Why did you close the Pull Request?

**Comment by nancy43** [comment]
i did it by mistake , pulled the request again

**Comment by caiodias** [comment]
Please, don't close the PR anymore.

**Comment by nancy43** [comment]
ok

**Comment by caiodias** [comment]
You changed the root index.html https://github.com/ProDigi-Developement/bootcamp-group-web-project/pull/21/files#diff-eacf331f0ffc35d4b482f1d15a887d3b file, you should create your own index.html file under your student folder. On the root index.html file you only have to add your button that redirects to your index.html file under your student's folder.

**Comment by caiodias** [comment]
You still didn't do https://github.com/ProDigi-Developement/bootcamp-group-web-project/pull/21#issuecomment-400737135

**Comment by nancy43** [comment]
Can you please check it again?

File: css/custom.css [hunk 1]
```diff
@@ -1,50 +0,0 @@
-
```
Comment (caiodias) [inline css/custom.css]: Please don't delete the files that the example and main page are using.

File: index.html [hunk 2]
```diff
@@ -9,10 +9,8 @@
 
     <!-- Bootstrap -->
     <link href="css/bootstrap.min.css" rel="stylesheet">
-
```
Comment (caiodias) [inline index.html]: Please don't delete the files or code that the example and main page are using.

File: index.html [hunk 3]
```diff
@@ -9,10 +9,8 @@
...
-    <!-- Custom -->
-    <link href="css/custom.css" rel="stylesheet">
-
+    <link href="css/bootstrap-theme.min.css" rel="stylesheet">
```
Comment (caiodias) [inline index.html]: This isn't the place to add your own code. Create your own index.html file under your student folder.

**Comment by caiodias** [comment]
You have to revert the deletion of custom.css file on css folder.

**Comment by caiodias** [comment]
Well done

**Pull request merged**